packages/ckeditor5-source-editing/README.md

```diff
@@ -1,4 +1,20 @@
 CKEditor 5 source editing feature
-=================================
+========================================
```

Comment:
I'd revert the change. I guess the number of `=` should be equal to the length of the title string.

Reply:
Done.